src/auditwheel/tools.py

```diff
@@ -157,11 +176,14 @@ def __init__(
         required: bool = True,
         default: str | None = None,
         choices: Iterable[str] | None = None,
+        type: type | None = None,
         **kwargs: Any,
     ) -> None:
         self.env_default = os.environ.get(env)
         self.env = env
         if self.env_default:
+            if type:
+                self.env_default = type(self.env_default)
```

Comment:
can a test be added in tests/unit/test_tools.py in order to cover this line please ?